flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/JobManagerCommunicationFactory.java

```diff
@@ -27,15 +26,6 @@
 public interface JobManagerCommunicationFactory {
```

Comment:
This has been become obsolete with your change and is unused. Let's remove it completely?